modules/service/src/test/scala/lucuma/odb/graphql/DatabaseOperations.scala

```diff
@@ -1484,6 +1484,12 @@ trait DatabaseOperations { this: OdbSuite =>
       .use(_.prepareR(command).use(_.execute(role, tid).void))
   }
 
+  def setObservationCalibratioRole(oid: Observation.Id, role: CalibrationRole): IO[Unit] = {
```

Comment:
I think this was defined in `CalibrationService` already

Reply:
Ah. Can I call a service method from the tests?

Reply:
Sure, though I think it ought to be a static method